bin/data_to_model_score.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import sys
+
+
+def score_submission(predictions_path: str, status: str) -> dict:
```

Comment:
nit: Return type is dict, but tuple is being returned:
https://sonarcloud.io/project/issues?resolved=false&pullRequest=9&id=Sage-Bionetworks-Workflows_nf-synapse-challenge&open=AY3DvQ_BTWBSDso3Si0m